client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/components/site-address-field.tsx

```diff
@@ -0,0 +1,55 @@
+import { FormLabel } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { Controller, Control } from 'react-hook-form';
+import getValidationMessage from 'calypso/blocks/import/capture/url-validation-message-helper';
+import { CAPTURE_URL_RGX } from 'calypso/blocks/import/util';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+import { CredentialsFormData } from '../types';
+import { ErrorMessage } from './error-message';
+
+interface Props {
+	control: Control< CredentialsFormData >;
+	errors: any;
```

Comment:
Suggestion: Can we have the shape of the error object here?

Reply:
Addressed the comments in this PR https://github.com/Automattic/wp-calypso/pull/94502